src/components/tables/RowActions/Shared/types.ts

```diff
@@ -48,7 +48,7 @@ export interface NestedListItemProps {
 
 export interface RowActionConfirmationProps {
     message: any;
-    selected: any; //SelectableTableStore['selected'];
+    selected: string[] | RowConfirmation[]; //SelectableTableStore['selected'];
```

Comment:
These are the main two types we are expecting so marking it that way.

Leaving the old comment until we finish the `todo` called out elsewhere.